tests/frontend/test_versions.py

```diff
@@ -0,0 +1,139 @@
+
+
+@pytest.mark.nondestructive
+def test_ratings_card(selenium, base_url, variables):
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    page = Versions(selenium, base_url)
+    # verify if ratings card is present
+    assert page.rating_card
+
+
+@pytest.mark.nondestructive
+def test_license_link(selenium, base_url):
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    page = Versions(selenium, base_url)
+
+    for i in range(len(page.versions_list)):
+        if page.versions_list[i].license_link is not False:  # if link exists
+
+            if (
+                page.versions_list[i].license_link.text == 'Custom License'
+            ):  # if link exists and is 'Custom License'
+                page.versions_list[i].license_link.click()
+                page.wait.until(
+                    EC.visibility_of_element_located(
+                        (By.CLASS_NAME, 'AddonInfo-info-html')
+                    )
+                )
+
+            else:  # if link exists and is not 'Custom License'
+                expected_link = page.versions_list[i].license_link.get_attribute('href')
+                if 'http' or 'https' in expected_link:
+                    # full url example-> https://example.com/something/something
+                    # turn into-> example.com/something/something
+                    expected_link = expected_link.split('//')[1]
+                    # turn into-> example.com
+                expected_link = expected_link.split('/')[0]
+                page.versions_list[i].license_link.click()
+                assert expected_link in selenium.current_url
+
+            # in both cases, we go back
+            page.driver.back()
+            page.driver.refresh()
+        else:  # if link does not exist
+            assert 'All Rights Reserved' in page.versions_list[i].license_text
+
+
+@pytest.mark.nondestructive
+def test_current_version(selenium, base_url):
+    addon_url = 'addon/devhub-listed-ext-07-30/'
+    # get info from api
+    response = requests.get(f'{base_url}/api/v5/addons/{addon_url}')
+    addon_version = response.json()['current_version']['version']
+    addon_size_kb = (
+        str(round(response.json()['current_version']['file']['size'] / 1024, 2)) + ' KB'
+    )
+    api_date = response.json()['current_version']['file']['created'][:10]
+    api_creation_date = datetime.strptime(api_date, '%Y-%m-%d')
+    # verify info displayed in page
+    page = Versions(selenium, base_url)
+    selenium.get(f'{base_url}{ADDON_VERSION_PAGE}')
+    assert page.versions_list[0].version_number == addon_version
+    assert addon_size_kb == page.versions_list[0].version_size
+    frontend_date = page.versions_list[0].released_date
+    # Special case: if date is 'Dec 9, 2021' it should be 'Dec 09, 2021', so we must add that '0'
+    if frontend_date[5] is ',':
+        frontend_date = frontend_date[:4] + '0' + frontend_date[4:]
+    frontend_creation_date = datetime.strptime(frontend_date, '%b %d, %Y')
```

Comment:
Why do you need to add the `0` manually? Using this conversion should already match what we have on the frontend for version date:  `api_creation_date.strftime('%b %#d, %Y')`. See https://strftime.org for reference.